src/app/test-record/test-record.component.html

```diff
@@ -1,4 +1,76 @@
-<p>
-  <span>JSON response:</span>
-  {{ testType | json }}
-</p>
+<div class="govuk-width-container">
+  <a class="govuk-back-link" vtmBackButton>Back</a>
```

Comment:
probably one for later but I think this should be part of the shell component which should contain
the navigation, header, container (showing tech-record and test-result sections)
I know this was in place prior to ADR i believe so this will need some attention but will require some time to carefully move the right pieces around.